### DISCUSSION TITLE
MC Server信息主页

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#4991] [讨论]
~~aaahhhh~~
![QQ截图20241020200852](https://github.com/user-attachments/assets/0b5d24d3-265c-4868-8fd1-1b32530ff567)
可喜可贺（
终于你能在PCL获取到实时的服务器信息了（

基于api.mcsrvstat.us API 

国内访问可能需要换个API...

CF WORKER构建 
你可以通过此链接获取嗨皮咳嗽的实时人数

新闻是空的 因为没API获取个空气啊 你可以自己改改（（（（
```
https://mcserver.plaincraftlauncher2.workers.dev/
```
UI By Openbmclapi主页

~~（或者学我一样 当MC运行就跑脚本killPCL强制刷新一次（人就不用动了 东西也能看到~~

关于源码
这个 如果有大佬愿意修一下Motd获取那就更好了...

CF可用 直接复制过去就行
```
https://github.com/Joker2184/MCS-GET-PCL2/blob/main/GET.js
```

**Comment by Joker2184** [comment]


**Reply by hejiehao** [reply]
……？

**Reply by Joker2184** [reply]
> ……？

差不多就API一直获取服务器信息然后再转到客户端（

**Comment by Joker2184** [comment]
嘶 选了个国内用不了的api .jpg
有人能推荐吗

**Reply by llsgllsg** [reply]
https://www.mczfw.cn/api 这个可以用
